### ISSUE TITLE
Remove current issue link from the cover image on the homepage

### ISSUE THREAD

**Issue by israelcefrin** [pkp/pkp-lib#5195]
When the JM sets current issue to be displayed on the homepage it shows the title,a cover image and a ToC to all published articles for that issue.

It is not necessary create a additional link to get to the same content than home.
We should remove the image cover link to the current issue. 

The code that is parsed on homepage of the journal should be change from this:
```
<div class="heading">
<a class="cover" href="https://publicknowledgeproject.org/index.php/publicknowledge/issue/view/2">
<img src="https://.publicknowledgeproject.org/public/journals/1/cover_issue_2_en_US.jpg">
</a>
[...]
</div>
```

To this:
```
<div class="heading">
<img src="https://.publicknowledgeproject.org/public/journals/1/cover_issue_2_en_US.jpg">
[...]
</div>
```

**Comment by israelcefrin** [comment]
@NateWr I was reviewing the code and neither homepage nor current issue page needs a link to the issue since they are already showing the list of articles. However, I think we could add a link to the list of articles on the same page. 

The code would turn from this :
`<a class="cover" href="https://publicknowledgeproject.org/index.php/publicknowledge/issue/view/2">`

To this:
`<a class="cover" href="#articles">`

And we could add the `id` attribute to the component in the `issue_toc.tpl` file. E.g.:

```
	{* Articles *}
	<div class="sections" id="articles" >
```

We can keep the same code that will work on index or issue page.

**Comment by NateWr** [comment]
I don't think this will improve things. It doesn't seem very intuitive that clicking on the issue cover image will take me down the page to look at articles. With an anchor link, the screen will skip down the page, which can be jarring.

I don't think the articles are very far away, so I'd say let the user scroll. It will be just as quick and less confusing.

**Comment by israelcefrin** [comment]
Given that the file `issue_toc.tpl` is used on requested pages: index and issue, do you think we can remove the link from the cover image on both? 
I was testing a conditional for checking if it was "index". but at end of the day, the "issue" page doesn't need a link to itself too.

**Comment by NateWr** [comment]
Yes, I think a link doesn't make sense on either page, so it can be removed without any conditions.

**Comment by israelcefrin** [comment]
PR:
https://github.com/pkp/ojs/pull/3485
